xgo/ios-simulator/Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM statusteam/xgo:1.9.2
 
+ADD update_ios.sh /update_ios.sh
+ENV UPDATE_IOS /update_ios.sh
```

Comment:
Why this definition as environment variable?

Reply:
I have no idea. The original image defines it this way. I think it should be fine without env in this case but I just followed the pattern.